Maple2.Server.Game/Session/GameSession.cs

```diff
@@ -441,7 +441,9 @@ public sealed partial class GameSession : Core.Network.Session {
 #endif
             Guild.Dispose();
             Buddy.Dispose();
-            Party.Dispose();
+            if (!Party.CheckDisband(CharacterId)) {
```

Comment:
I think `Dispose()` should always happen as it's cleaning up resources, maybe `CheckDisband` in `Dispose()`?